io-engine/src/core/device_monitor.rs

```diff
@@ -65,12 +65,14 @@ pub async fn device_monitor_loop() {
 
                     match rx {
                         Err(e) => {
-                            error!(
-                                "Failed to schedule removal request: {}",
-                                e.verbose()
-                            )
+                            error!("Failed to schedule removal request: {e}")
+                        }
+                        Ok(rx) => {
+                            if let Err(e) = rx.await {
+                                error!("Error processing removal request: {e}");
```

Comment:
isn't error just cancelled here? Might be more useful to add nexus name and child device?